netci.groovy

```diff
@@ -1432,74 +1501,123 @@ combinedScenarios.each { scenario ->
                                     buildNumber('${CORECLR_WINDOWS_BUILD}')
                                 }
                             }
+
+                            if (scenario == 'coverage') {
+                                // Move coreclr to clr directory
+                                shell("rm -rf .clr; mkdir .clr; mv * .clr; mv .git .clr; mv .clr clr")
+                                // Get corefx
+                                shell("git clone https://github.com/dotnet/corefx fx")
+                                shell("cd fx")
+                                // Build Linux corefx
+                                shell("./build.sh x64 release Linux skiptests")
+                                // Check contents of bin directory - this can be removed after we confirm everything is as expected
+                                shell("ls bin")
+                                shell("cd ${WORKSPACE}")
+
+                                // Run corefx tests
+                                shell("""./fx/run-test.sh \\
+                --coreclr-bins ${WORKSPACE}/clr/bin/Product/${osGroup}.${architecture}.${configuration} \\
+                --mscorlib-bins ${WORKSPACE}/clr/bin/Product/${osGroup}.${architecture}.${configuration} \\
+                --corefx-tests ${WORKSPACE}/fx/bin/tests/{osGroup}.AnyCPU.{configuration} \\
+                --corefx-native-bins ${WORKSPACE}/fx/bin/${osGroup}.${architecture}.${configuration}""")
+
+                                // Run coreclr GC tests w/ server GC enabled
+                                shell("""./clr/tests/runtest.sh \\
+                --testRootDir=\"\${WORKSPACE}/clr/bin/tests/Windows_NT.${architecture}.${configuration}/GC\" \\
```

Comment:
This won't run the long-running GC tests, I think - we'll probably miss out on coverage for low-memory GC situations. Right now the long GC test job makes (ab)use of the --runFailingTestsOnly option to run a subset of tests - I could easily split the long GC tests into a separate "failing tests" file so that we could run the long GC tests in one invocation here, if that's what we're trying to do.